### PR TITLE
ndd开源组织.Notepad-- version 1.22.2

### PR DESCRIPTION
**Pull request by rtrobin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/95958)

### CODE REVIEW

**Comment by rtrobin** [comment]
@microsoft-github-policy-service agree

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ndd开源组织.Notepad--
+PackageVersion: 1.22.1
+PackageLocale: en-US
+Publisher: ndd开源组织
+PublisherUrl: https://github.com/cxasm/notepad--
+PublisherSupportUrl: https://github.com/cxasm/notepad--
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:9]: Support URL should not be the same as Package URL

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ndd开源组织.Notepad--
+PackageVersion: 1.22.1
+PackageLocale: en-US
+Publisher: ndd开源组织
+PublisherUrl: https://github.com/cxasm/notepad--
+PublisherSupportUrl: https://github.com/cxasm/notepad--
+PrivacyUrl: https://github.com/cxasm/notepad--
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:10]: Privacy URL should not be the same as PackageURL

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: ndd开源组织.Notepad--
+PackageVersion: 1.22.1
+PackageLocale: en-US
+Publisher: ndd开源组织
+PublisherUrl: https://github.com/cxasm/notepad--
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:8]: Publisher URL should not be the same as Package URL

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+PackageLocale: en-US
+Publisher: ndd开源组织
+PublisherUrl: https://github.com/cxasm/notepad--
+PublisherSupportUrl: https://github.com/cxasm/notepad--
+PrivacyUrl: https://github.com/cxasm/notepad--
+Author: cxasm
+PackageName: Notepad--
+PackageUrl: https://github.com/cxasm/notepad--
+License: GPL-3.0 license
+Copyright: https://github.com/cxasm/notepad--/blob/main/LICENSE
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:15]: This needs to be the actual copyright text, not a link to it

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+CopyrightUrl: https://github.com/cxasm/notepad--/blob/main/LICENSE
+ShortDescription: Notepad-- Setup
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:17]: Please provide a better description of what Notepad-- does

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+Description: https://github.com/cxasm/notepad--/blob/main/LICENSE
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:18]: Description should not be a URL

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml [hunk 7]
```diff
@@ -0,0 +1,21 @@
...
+ReleaseNotesUrl: https://github.com/cxasm/notepad--/releases
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.locale.en-US.yaml:19]: This should be a direct link to the release notes for this version

File: manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.installer.yaml [hunk 8]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: ndd开源组织.Notepad--
+PackageVersion: 1.22.1
+Installers:
+- InstallerUrl: https://github.com/cxasm/notepad--/releases/download/Notepad--v1.22/Notepad--v1.22.1-Installer.exe
+  Architecture: x64
+  InstallerType: inno
+  InstallerSha256: CBC55446BE8C03395CCC8E9FF2C3EDEC3F23F2DDD314230819653FA1EE59F2CA
```
Comment (Trenly) [inline manifests/n/ndd开源组织/Notepad--/1.22.1/ndd开源组织.Notepad--.installer.yaml:10]:
```suggestion
Installers:
- InstallerUrl: https://github.com/cxasm/notepad--/releases/download/Notepad--v1.22/Notepad--v1.22.1-Installer.exe
  Architecture: x64
  InstallerType: inno
  InstallerSha256: CBC55446BE8C03395CCC8E9FF2C3EDEC3F23F2DDD314230819653FA1EE59F2CA
  Scope: machine
  InstallerSwitches:
    Custom: /ALLUSERS
- InstallerUrl: https://github.com/cxasm/notepad--/releases/download/Notepad--v1.22/Notepad--v1.22.1-Installer.exe
  Architecture: x64
  InstallerType: inno
  InstallerSha256: CBC55446BE8C03395CCC8E9FF2C3EDEC3F23F2DDD314230819653FA1EE59F2CA
  Scope: user
  InstallerSwitches:
    Custom: /CURRENTUSER
```

**Review by Trenly — CHANGES_REQUESTED** [review]

**Review by Trenly — APPROVED** [review]

**Pull request merged**